documentation/2.0/content/release-notes/_index.md

```diff
@@ -1,4 +1,58 @@
-### Known issues for WebLogic Deploy Tooling
++++
+title = "Release Notes"
+date = 2019-02-22T15:27:38-05:00
+weight = 5
+pre = "<b> </b>"
++++
+
+Review the following release notes for recent changes to WebLogic Deploy Tooling and known issues.
+
+- WebLogic Deploy Tooling now incorporates the SnakeYAML parser for reading and writing model files.
+This may require some changes to existing models in order to be parsed correctly.
+   - Model elements that use [delete notation]({{< relref "/concepts/model#declaring-named-mbeans-to-delete" >}}) need to be escaped in single or double quotation marks.
+   ```yaml
+   topology:
+       Server:
+           '!ms1':
+           ms2:
+   ```
+
+
+   - Model elements under the same parent should be indented to the exact same level. The previous YAML parser did not enforce this restriction,
+   but it is standard for YAML. In this example, each cluster is indented four spaces.
+   ```yaml
+   topology:
+       Cluster:
+           cluster1:
+               ClientCertProxyEnabled: True
+           cluster2:
+               WeblogicPluginEnabled: true
+   ```
+
+
+
+- Object lists in the `kubernetes` section of the model now should be specified in a hyphenated list format,
+similar to how they appear in the domain resource file produced for [WebLogic Kubernetes Operator](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/domain-resource/).
+```yaml
+    clusters:
+    - clusterName: 'cluster1'
+      allowReplicasBelowMinDynClusterSize: true
+    - clusterName: 'cluster2'
+      allowReplicasBelowMinDynClusterSize: true
+```
+- The "named object list" format is deprecated now, and will cause warning messages to be displayed.
```

Comment:
Can we make this a nested bullet under "Object lists in the ..." ?

Reply:
Sure (in the next commit).